core/src/components/card/card.tsx

```diff
@@ -82,6 +82,11 @@ export class Card implements ComponentInterface, AnchorInterface, ButtonInterfac
    */
   @Prop() routerAnimation: AnimationBuilder | undefined;
 
+  /**
+   * Set to `"soft"` for a card with more rounded corners.
+   */
+  @Prop({ reflect: true }) shape?: 'soft' | 'round' | 'rectangular';
```

Comment:
Do we need to add the `soft` shape? If soft is the default then we would just need to add `round` and `rectangular`:

<img width="632" alt="Screenshot 2024-04-08 at 11 39 50 AM" src="https://github.com/ionic-team/ionic-framework/assets/6577830/ecdc012f-b85e-4074-8b6a-9936fc5c6e87">

Reply:
Yes, we can change the behavior of card shape.

Done ✅

Reply:
And what if in the future we want to change the default for some reason, to rectangular? Or any other value (for some strange reason 😂 ). We would need to change the public API as well. 

In my opinion the shape options and the default shape should be separate things, even if in the present they share the same value.

Reply:
I have the same opinion and that was the rational of implemented the soft shape. @brandyscarney what is your opinion? Can we revert this to have the 3 shape, even if the default and soft has the same border-radius?

Reply:
This is a great point about changing the default in the future introducing changes to the public API. I think we should stick with matching the [button implementation](https://github.com/ionic-team/ionic-framework/blob/7ce115993aa2be4ba70881e5767e4ffd3fedfe21/core/src/components/button/button.tsx#L120) for now in order to get this merged in. Then, I have created a ticket [FW-6118](https://outsystemsrd.atlassian.net/browse/FW-6118) to investigate setting the default for the components that have added a `shape` property (button, card, input, etc). 

We will need to do some investigation here to make sure that we have consistent behavior among components, and also check that it doesn't cause unexpected behaviors for developers trying to override styles. Does that work for now?